### PR TITLE
Sprint1

### PR DESCRIPTION
**Pull request by philipp-bode**

### CODE REVIEW

File: src/lib/storage/storage_manager.cpp [hunk 1]
```diff
@@ -10,38 +10,33 @@
...
 
-void StorageManager::add_table(const std::string& name, std::shared_ptr<Table> table) {
-  // Implementation goes here
-}
+void StorageManager::add_table(const std::string& name, std::shared_ptr<Table> table) { _tables[name] = table; }
 
-void StorageManager::drop_table(const std::string& name) {
-  // Implementation goes here
-}
+void StorageManager::drop_table(const std::string& name) { _tables.erase(name); }
```
Comment (m0r13) [inline src/lib/storage/storage_manager.cpp]: According to the tests this is supposed to throw an exception. But of course with a `map` it doesn't. What to do here guys? @Philipp-Bode @nilsthamm

Comment (philipp-bode) [reply src/lib/storage/storage_manager.cpp]: Probably just check against the number of deleted elements and throw manually, right?
Not necessarily super pretty, but OK enough, I'd say.

Comment (m0r13) [reply src/lib/storage/storage_manager.cpp]: Okay. sec.

Comment (m0r13) [reply src/lib/storage/storage_manager.cpp]: Done.

File: src/lib/storage/table.cpp [hunk 2]
```diff
@@ -17,53 +17,63 @@
...
-  return ChunkID{0};
-}
+ChunkID Table::chunk_count() const { return static_cast<ChunkID>(_chunks.size()); }
 
 ColumnID Table::column_id_by_name(const std::string& column_name) const {
-  // Implementation goes here
-  return ColumnID{0};
+  auto const pos = std::find(_column_names.begin(), _column_names.end(), column_name);
+  if (pos == _column_names.end()) {
+    throw std::runtime_error("Implement Table::get_chunk");
```
Comment (m0r13) [inline src/lib/storage/table.cpp]: What's with this one?

Comment (philipp-bode) [reply src/lib/storage/table.cpp]: Aye, good call, didn't change the exception message.

**Pull request merged**